Plugin/Magento/Checkout/CustomerData/Cart.php

```diff
@@ -123,6 +123,10 @@ public function afterGetSectionData(CustomerDataCart $subject, array $customerDa
         $customerData['boltCartHints'] = null;
         if ($quote->getId()) {
             try {
+                if ($this->boltHelperCart->getOrderByQuoteId($quote->getId())) {
```

Comment:
Are we sure that the order is paid? It can be an incomplete order after the wrong payment that we are going to delete
CC: @garmashdev

Reply:
Sounds good to me